### PR TITLE
Add GTIN-14 formatting

### PR DESCRIPTION
**Pull request by kokoshneta**
- Add GTIN-14 formatting
- Add use case for the readme
- Move descriptive comments above their respective class properties

### CODE REVIEW

File: src/Biblys/Isbn/Isbn.php [hunk 1]
```diff
@@ -353,6 +381,11 @@ private function setAgency($agency)
         $this->_agency = $agency;
     }
 
+    private function setPrefix($prefix)
+    {
+        $this->_prefix = $prefix;
+    }
+
```
Comment (iwazaru) [inline src/Biblys/Isbn/Isbn.php:388]: On second thought, could you rename the method and property everywhere to something like `gtin14Prefix` so there's not confusion with xml ranges prefixes or any future other kind of prefix?

File: src/Biblys/Isbn/Isbn.php [hunk 2]
```diff
@@ -21,29 +21,32 @@ class Isbn
...
-    private $_prefixes;
+    private $_format = 'EAN';
     // XML ranges file prefixes
-    private $_groups;
+    private $_prefixes;
     // XML ranges file groups
-            private $_agency; // ISBN Agency
+    private $_groups;
+    // ISBN Agency
+    private $_agency;
```
Comment (iwazaru) [inline src/Biblys/Isbn/Isbn.php:49]: Yay! Don't know what I was think putting comments under the code…

File: tests/IsbnTest.php [hunk 3]
```diff
@@ -57,6 +57,12 @@ public function testFormatIsbn10()
         $this->assertEquals($isbn10->format('ISBN-10'), "3-464-60352-0");
     }
 
+	public function testFormatGtin14()
+	{
+		$isbn = new Isbn('9783464603529');
+		$this->assertEquals($isbn->format('GTIN-14'), '19783464603526');
+	}
```
Comment (iwazaru) [inline tests/IsbnTest.php:64]: You can run test locally by running `composer install` (once) and then `composer test`. You'll notice that your test is passing but your changes have broken some ISBN-13 related tests.

Also CI was broken when your opened your pull request but if [you rebase your branch](https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request) against master or develop, they will be run by Github when you push.

**Review by iwazaru — CHANGES_REQUESTED** [review]

File: src/Biblys/Isbn/Isbn.php (hunk 1 above)
Comment (kokoshneta) [reply src/Biblys/Isbn/Isbn.php]: Absolutely – done!

File: tests/IsbnTest.php (hunk 3 above)
Comment (kokoshneta) [reply tests/IsbnTest.php:64]: Got it figured out and rebased! Also figured out the logical error that made the ISBN-13 formatting go haywire. I’d got my odds and evens mixed up when moving from a 13-item array to a 14-item one, so now the array gets reversed and then the new key is used instead.

**Comment by kokoshneta** [comment]
It seems just pushing my post-review changes automatically added them to the pull request – does everything look right now?

File: src/Biblys/Isbn/Isbn.php [hunk 4]
```diff
@@ -311,9 +332,19 @@ private function calculateChecksum($format = 'EAN')
...
+
+            foreach ($c as $k => $v) {
+                if ($k & 1) { // If current array key is odd
+                    $sum += $v;
+                } else { // If current array key is even
+                    $sum += $v * 3;
+                }
+            }
+
+            \var_dump($sum);
```
Comment (iwazaru) [inline src/Biblys/Isbn/Isbn.php:346]: You forgot a `var_dump` here :)

Comment (kokoshneta) [reply src/Biblys/Isbn/Isbn.php:346]: Oh ferchrissakes, I’d forget my brain if it weren’t screwed on to the inside of my head!

Comment (kokoshneta) [reply src/Biblys/Isbn/Isbn.php]: I also forgot to add a PHPDoc line for the new parameter `$prefix` to the `format()` function (fixed now).

**Review by iwazaru — APPROVED** [review]

**Pull request merged**